zipkin-server/src/test/java/zipkin2/server/internal/ITZipkinGrpcCollector.java

```diff
@@ -80,30 +79,33 @@ class ITZipkinGrpcCollector {
   }
 
   @Test void report_emptyIsOk() throws IOException {
-
     callReport(new ListOfSpans.Builder().build());
   }
 
-  ReportResponse callReport(ListOfSpans spans) throws IOException {
-    Buffer requestBody = new Buffer();
-    requestBody.writeByte(0 /* compressedFlag */);
-    Buffer encodedMessage = new Buffer();
-    ListOfSpans.ADAPTER.encode(encodedMessage, spans);
-    requestBody.writeInt((int) encodedMessage.size());
-    requestBody.writeAll(encodedMessage);
-
-    Response response = client.newCall(new Request.Builder()
-      .url(url(server, "/zipkin.proto3.SpanService/Report"))
-      .addHeader("te", "trailers")
-      .post(RequestBody.create(requestBody.snapshot(), MediaType.get("application/grpc")))
-      .build())
-      .execute();
-
-    BufferedSource responseBody = response.body().source();
-    assertThat((int) responseBody.readByte()).isEqualTo(0); // uncompressed
-    long encodedLength = responseBody.readInt() & 0xffffffffL;
-
-    return ReportResponse.ADAPTER.decode(responseBody);
+  void callReport(ListOfSpans spans) throws IOException {
```

Comment:
I refactored this to close resources.. only other change is verifying the encodedLength is the same as the input. Since the result has no fields, I changed to void return